tests/unit/test_config.py

```diff
@@ -105,7 +108,10 @@ def test_ne_update_s3_storage_config(self):
     def test_build_memberlist_config(self):
         self.cluster_provider.gather_addresses.return_value = ["address1", "address2"]
         memberlist_config = self.coordinator._build_memberlist_config()
-        expected_config = {"join_members": ["address1", "address2"]}
+        expected_config = {"cluster_label": "something", "join_members": ["address1", "address2"]}
+        self.assertIn("cluster_label", expected_config)
+        memberlist_config.pop("cluster_label")
+        expected_config.pop("cluster_label")
```

Comment:
Can we assert some dummy labels instead?

Reply:
Yes, and I'm changing that; however the function is still gonna look a bit awkward; this is because we are patching out the entire `_cluster_provider`, and the `cluster_label` comes from some properties nested deep inside that object (`.cluster_provider._charm.model.name`, etc...)